recipes/app2.rb

```diff
@@ -82,8 +101,8 @@
     pid_file '/home/formsender-staging/tmp/pids/gunicorn.pid'
     exec_start '/home/formsender-staging/venv/bin/gunicorn -b 0.0.0.0:8086 '\
       '-D --pid /home/formsender-staging/tmp/pids/gunicorn.pid '\
-      '--access-logfile /var/log/formsender-staging_access.log '\
-      '--error-logfile /var/log/formsender-staging_error.log '\
+      '--access-logfile /home/formsender-staging/logs/formsender-staging_access.log '\
+      '--error-logfile /home/formsender-staging/logs/formsender-staging_error.log '\
```

Comment:
I don't think we need to duplicate the names of the log files with the instance that's running. I recommend you just go with ``access.log`` and ``error.log`` for each service.